clion-plugin/src/main/kotlin/org/utbot/cpp/clion/plugin/ui/targetsToolWindow/UTBotTargetsController.kt

```diff
@@ -15,7 +16,7 @@ import org.utbot.cpp.clion.plugin.utils.logger
 import testsgen.Testgen
 
 @Service
-class UTBotTargetsController(val project: Project) {
+class UTBotTargetsController(val project: Project): Disposable {
```

Comment:
Actually I do not understand why an instance of the controller should be disposable...

Reply:
ok, I agree. Changed disposable to `ClientManager`. Any disposable with project lifetime (project service for example) will do